runelite-api/src/main/java/net/runelite/api/Varbits.java

```diff
@@ -297,6 +297,13 @@
 	PERSONAL_POINTS(5422),
 	RAID_PARTY_SIZE(5424),
 
+	/**
+	 * Braziers
+	 */
+	BRAZIER_6532(6532),
+	BRAZIER_6533(6533),
+	BRAZIER_6544(6544),
```

Comment:
These names should contain some information about where the brazier is, instead of just a number